core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/Recommended.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Adds a validation that the field or a file is recommended.
+ *
+ * <p>Example.
+ *
+ * <pre>
+ *   {@literal @}GtfsTable(value = "feed_info.txt", singleRow = true) {
+ *   @Recommended
```

Comment:
My bad, there was an error in my previous suggestion. It makes the Task :aggregateJavadoc to fail [here](https://github.com/MobilityData/gtfs-validator/runs/6619597119?check_suite_focus=true) and [here](https://github.com/MobilityData/gtfs-validator/runs/6619597193?check_suite_focus=true).
```suggestion
 *   {@literal @}Recommended
```